_sass/custom/custom.scss

```diff
@@ -1,4 +1 @@
-$pink-000: #f77ef1;
-$pink-100: #f967f1;
-$pink-200: #e94ee1;
-$pink-300: #dd2cd4;
+// custom CSS rules goes here
```

Comment:
```suggestion
// custom SCSS rules go here
```

Reply:
I'll provide an alternate more specific text; I did mean CSS rules (as in the formal spec definition)

Reply:
But the file extension is `scss`, so I think Jekyll will compile it anyway?

Reply:
Oh, the idea wasn't to say that only CSS is allowed; rather, that everything in that file should emit CSS (after compilation), rather than just being variable declarations.

I think I'll revert this change somewhat since that's not actually a constraint.